easybuild/easyconfigs/s/subread/subread-1.6.5-GCC-8.3.0.eb

```diff
@@ -0,0 +1,36 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'subread'
+version = '1.6.5'
```

Comment:
`2.0.0` doesn't work as dep for `BinSanity`?

Reply:
Unfortunately not: only 1.x.x - https://github.com/edgraham/BinSanity/wiki/Installation#installation-of-dependencies
> *Install featureCounts - To do this download the latest subread package (subread-1.x.x-source.tar.gz)*

Reply:
Tried 2.0.0 and test works - someone will need to try using it more. I will include both versions for Subread so we can switch if needed?